homeassistant/components/vacuum/roomba.py

```diff
@@ -5,17 +5,16 @@
 https://home-assistant.io/components/vacuum.roomba/
 """
 import asyncio
+import homeassistant.helpers.config_validation as cv
```

Comment:
That's some creative "sorting". I'm used to selecting all imports in vim and doing `:sort`.

The [style guide](https://home-assistant.io/developers/development_guidelines/) simply states they should be "ordered". 

Anyway I'll move this line down.